routes/web.php

```diff
@@ -31,4 +31,9 @@
     Route::get('tours/{id}', 'HomeController@tourDetail')->name('tour.detail');
     Route::view('errors', 'book_tour.pages.errors')->name('errors');
     Route::get('categories/{id}/tours', 'HomeController@tourByCategory')->name('tour.category');
+//-------------auth---------------------------------------------
+    Route::get('register', 'UserController@getRegister')->name('register.get');
+    Route::post('register', 'UserController@postRegister')->name('register.post');
+    Route::get('login', 'UserController@getLogin')->name('login.get');
+    Route::post('login', 'UserController@postLogin')->name('login.post');
```

Comment:
cố gắng đặt tên hàm và route name có ý nghĩa và dễ hiểu
```suggestion
    Route::get('register', 'UserController@getFormRegister')->name('register.get');
    Route::post('register', 'UserController@register')->name('register.post');
    Route::get('login', 'UserController@getFormLogin')->name('login.get');
    Route::post('login', 'UserController@login')->name('login.post');
```